src/pages/workspace/invoices/WorkspaceInvoiceVBASection.tsx

```diff
@@ -0,0 +1,270 @@
+                    title: account?.addressName ?? '',
+                    icon,
+                    description: PaymentUtils.getPaymentMethodDescription(accountType, account),
+                    type: CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT,
+                };
+            }
+            setPaymentMethod({
+                isSelectedPaymentMethodDefault: transferBankAccountID === methodID,
+                selectedPaymentMethod: account ?? {},
+                selectedPaymentMethodType: accountType,
+                formattedSelectedPaymentMethod,
+                methodID: methodID ?? '-1',
+            });
+            setShouldShowDefaultDeleteMenu(true);
+            setMenuPosition();
+            return;
+        }
+        setShouldShowAddPaymentMenu(true);
+        setMenuPosition();
+    };
+
+    /**
+     * Hide the add payment modal
+     */
+    const hideAddPaymentMenu = () => {
+        setShouldShowAddPaymentMenu(false);
+    };
+
+    /**
+     * Hide the default / delete modal
+     */
+    const hideDefaultDeleteMenu = useCallback(() => {
+        setShouldShowDefaultDeleteMenu(false);
+        setShowConfirmDeleteModal(false);
+    }, [setShouldShowDefaultDeleteMenu, setShowConfirmDeleteModal]);
+
+    const deletePaymentMethod = useCallback(() => {
+        const bankAccountID = paymentMethod.selectedPaymentMethod.bankAccountID;
+        if (paymentMethod.selectedPaymentMethodType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT && bankAccountID) {
+            BankAccounts.deletePaymentBankAccount(bankAccountID);
+        }
+    }, [paymentMethod.selectedPaymentMethod.bankAccountID, paymentMethod.selectedPaymentMethodType]);
+
+    const makeDefaultPaymentMethod = useCallback(() => {
+        // Find the previous default payment method so we can revert if the MakeDefaultPaymentMethod command errors
+        const paymentMethods = PaymentUtils.formatPaymentMethods(bankAccountList ?? {}, {}, styles);
+        const previousPaymentMethod = paymentMethods.find((method) => !!method.isDefault);
+        const currentPaymentMethod = paymentMethods.find((method) => method.methodID === paymentMethod.methodID);
+        if (paymentMethod.selectedPaymentMethodType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT) {
+            PaymentMethods.setInvoicingTransferBankAccount(currentPaymentMethod?.methodID ?? -1, policyID, previousPaymentMethod?.methodID ?? -1);
+        }
+    }, [bankAccountList, styles, paymentMethod.selectedPaymentMethodType, paymentMethod.methodID, policyID]);
+
+    /**
+     * Navigate to the appropriate payment type addition screen
+     */
+    const addPaymentMethodTypePressed = (paymentType: string) => {
+        hideAddPaymentMenu();
+        if (paymentType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT || paymentType === CONST.PAYMENT_METHODS.BUSINESS_BANK_ACCOUNT) {
+            BankAccounts.openPersonalBankAccountSetupView();
+            return;
+        }
```

Comment:
As discussed in https://github.com/Expensify/App/issues/68031, we decided to only allow connecting BBA inside Workflows.

https://github.com/Expensify/App/issues/68031#issuecomment-3365865082